src/app/users/create-user/create-user.component.ts

```diff
@@ -0,0 +1,810 @@
+import { Component, OnInit, ViewChild } from '@angular/core';
+import { Router } from '@angular/router';
+import { FormioComponent } from '@formio/angular';
+import { UsersService } from 'src/app/services/users.service';
+import { IUserItem } from '../user-item-model';
+import { MdbModalRef, MdbModalService } from 'mdb-angular-ui-kit/modal';
+
+@Component({
+  selector: 'app-create-user',
+  templateUrl: './create-user.component.html',
+  styleUrls: ['./create-user.component.scss']
+})
+export class CreateUserComponent implements OnInit {
+
+  @ViewChild(FormioComponent, { static: false })
+  form!: FormioComponent;
+  FormData: any = {
```

Comment:
I think this is too much just for create user form. we can simply use reactive form to get these data